src/config.js

```diff
@@ -0,0 +1,25 @@
+import { config } from 'dotenv';
+import debug from 'debug';
+
+config();
+
+const debugConfig = debug('config');
+
+const { PORT = '1337', HOST = 'localhost' } = process.env;
+
+//HACK: You need to tap $env:DEBUG="config" in terminal to turn on debug
```

Comment:
I don't feel that it's a hack. It's the normal way to enable debugging, so it's just a note 😄

Reply:
Ok) It was just funny to use HACK)